packages/terra-consumer-nav/src/components/user-profile/UserProfile.scss

```diff
@@ -47,7 +47,7 @@
   }
 
   .popup-button {
-    background: var(--terra-consumer-nav-background, rgba(255, 255, 255, 0.5));
+    background: var(--terra-consumer-nav-background, rgba(255, 255, 255, 0.25));
```

Comment:
I'm assuming this was correlated with some UX decision?

Reply:
yup, .25 is what the rest of the nav is using as the opacity. The profile link with .5 was drawing too much attention to it.